lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -46,12 +47,17 @@ function getDefinitionsToRun(allTestDefns, requestedIds, {invertMatch}) {
     console.log('Running ALL smoketests. Equivalent to:');
     console.log(usage);
   } else {
-    smokes = allTestDefns.filter(test => invertMatch !== requestedIds.includes(test.id));
+    smokes = allTestDefns.filter(test => {
+      // Include all tests that *include* requested id.
+      // e.g. a requested 'pwa' will match 'pwa-airhorner', 'pwa-caltrain', etc
+      const isRequested = requestedIds.some(requestedId => test.id.includes(requestedId));
+      return invertMatch !== isRequested;
```

Comment:
Since we're not trying to fit this on one line anymore, it could be more explicit.

```suggestion
	  if (invertMatch) isRequested = !isRequested;
      return isRequested;
```

Reply:
> Since we're not trying to fit this on one line anymore, it could be more explicit.

I like it